### ISSUE TITLE
Name of the repo/project

### ISSUE THREAD

**Issue by i2000s** [JuliaQuantum/QuCmp.jl#7]
I recently got a notification from @amitjamadagni regarding the naming conventions (in the chat). I would prefer to use the `.jl` as the suffix for the consistence with other coding project repos. Also, if everyone else is ok with the `QuCmp` prefix, I am fine. Otherwise, I think `QuComputing` would be more intuitive. Back in the day, @jrevels and I came out the idea to use `JuliaQ` as the name for our org, but people from the Julia community advised us to adopt an intuitive name in the end. 

@Roger-luo Feel free to announce this new project on our website, with a News post and adds to the project collection page.

@JuliaQuantum/owners decision please :smile_cat:

**Comment by i2000s** [comment]
@acroy Let me know if you are against the change of Repo's name to QuCmp.jl at this stage. I will solve other PRs before committing the change of name. Then we can merge another PR for the website statement hopefully within a week. We can think about other better names later.

**Comment by acroy** [comment]
No problem. QuCmp sounds good.

**Comment by amitjamadagni** [comment]
I guess the only change required is appending `.jl`to the package name. Once that is done we could merge the PR's. @i2000s

**Comment by Roger-luo** [comment]
It seems to be a naming rule for Julia packages' repo (I mean add a `.jl`) @i2000s

**Comment by i2000s** [comment]
@amitjamadagni I have double checked the GitHub doc, the renaming won't affect the current PRs. Therefore, I will commit the renaming right away. Sorry for the delay. 

@all, please fix your git remote url and page links for this repo following [this instruction](https://help.github.com/articles/renaming-a-repository/). We will use `QuCmp.jl` as the new repo name for now. Comment below if you find any related issues.
